src/client/app/components/about.ts

```diff
@@ -1,6 +1,10 @@
-import { Component, Input, ChangeDetectionStrategy, Output, EventEmitter, AfterViewInit } from '@angular/core';
+import { Component, Input, ChangeDetectionStrategy, Output, EventEmitter, AfterViewInit, ChangeDetectorRef } from '@angular/core';
 import { environment, storageSize, storage } from '../helpers';
 import {Strings, getAvailableLanguages, getDisplayLanguage, setDisplayLanguage } from '../strings';
+import { UIEffects } from '../effects/ui';
+const PLAYGROUND_ORIGIN = 'playground_origin_environment_url';
```

Comment:
There is a duplicate here in constants between this file and the `env.config.js` file. For some reason, the build was failing when I tried to import the constants in `about.ts` via the line

`const { PLAYGROUND_ORIGIN, PLAYGROUND_REDIRECT } = require('../../../../config/env.config');`

The pathname was correct, but it would fail. If there's a better way to do it, would welcome thoughts.